.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ '**' ]
```

Comment:
nit: think we can update to https://github.com/yext/analytics/blob/main/.github/workflows/run_tests.yml so that it runs on everything?